example.c

```diff
@@ -48,25 +91,24 @@ static ssize_t read_attr(const char *device, const char *attr,
 }
 
 static ssize_t write_attr(const char *device, const char *attr,
-		const char *buf, size_t len, bool debug)
+			  const char *buf, size_t len, bool debug)
 {
 	return -ENOSYS;
 }
-
+int32_t scale = 0;
```

Comment:
this `scale` variable could be made `static` and moved somewhere in the first part of the file, before the function definitions